packages/app/src/runner/useRunnerStyle.ts

```diff
@@ -57,12 +59,14 @@ export const useRunnerStyle = ({
   const autStore = useAutStore()
 
   const viewportStyle = computed(() => {
-    let scale: number = 1
+    let scale = 1
 
     if (!screenshotStore.isScreenshotting) {
       scale = Math.min(containerWidth.value / autStore.viewportDimensions.width, containerHeight.value / autStore.viewportDimensions.height, 1)
     }
 
+    autStore.setScale(scale)
```

Comment:
Triggering a side effect with `computed` is probably not ideal, they should generally be pure, computed data.

If we need it other places, how about moving this to the aut store, like this: https://github.com/cypress-io/cypress/pull/20068

Reply:
I agree with this. Something is up on that branch you linked, but in the short term I'll pull the side effect into a watcher for clarity, and move things into the aut-store in the next PR coming right up.